patterns/1-initial/good-first-project.md

```diff
@@ -0,0 +1,88 @@
+## Title
+
+Good First Project
+
+## Patlet
+
+An InnerSource program has been launched at an organization, and to get off to a successful start it requires some good first projects that lend themselves to InnerSource-style development. Assessing the InnerSource Fitness of the candidate projects can help to select projects that have the potential to help demonstrate the power of InnerSource.
+
+## Problem
+
+When launching an InnerSource program, it is essential to convince the skeptics about the utility of InnerSource (see the [Start as Experiment](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/start-as-experiment.md) pattern).
+
+While any project has the potential to benefit from being developed in InnerSource style, some are more suitable candidates than others. But how do you find the most appropriate projects that have the best chance to be successful as InnerSource projects?
+
+Moreover, while previous works focused on the criteria for readiness for InnerSource adoption at an organizational level, there seems to be no concise and actionable guideline for InnerSource-readiness for individual projects.
+
+## Story (optional)
+
+## Context
+
+- An organization is starting or is in the early phase of InnerSource adoption, with a lot of skepticism about whether or not it'll work.
+- Selecting an inappropriate pilot project might lead to failure of the entire initiative (see the [Start as Experiment](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/start-as-experiment.md) pattern).
+- Otherwise, a team might be planning to start an InnerSource project but they do not know how prepared they are and if the project they are putting forward is appropriate or not.
+
+## Forces
+
+- Lack of knowledge about which projects stand to benefit the most from InnerSource development.
+- The team might think they are prepared for InnerSource, but might not have an idea about what challenges lie ahead and how prepared they actually are.
+- Unsure if the tools used by the project and the organizational processes are conducive to InnerSource style development.
+
+## Sketch (optional)
+
```

Comment:
Do you want to add a sample of the output of your tool here?

Any type of image is always nice in a pattern, makes it a bit easier on the eye.
Even better if the image helps to understand the pattern of course ;-)

Reply:
Do you think this will be ok? https://imgur.com/a/wO4EyJj

Reply:
Yeah absolutely. Looks great!
How large is that file? If it is too large we could embed it into the pattern directly from imgur.com if that is possible?

Reply:
I have added the link to the demo video in a separate section. I felt that the solution needs to presented first for the demo to make sense. Let me know if that's ok

Reply:
That works. I moved it around and shortened it a bit.
Closing this thread.